src/ln/channelmonitor.rs

```diff
@@ -1780,7 +1980,7 @@ impl ChannelMonitor {
 			let mut res = vec![local_tx.tx.clone()];
 			match self.key_storage {
 				Storage::Local { ref delayed_payment_base_key, ref prev_latest_per_commitment_point, .. } => {
-					res.append(&mut self.broadcast_by_local_state(local_tx, prev_latest_per_commitment_point, &Some(*delayed_payment_base_key)).0);
+					res.append(&mut self.broadcast_by_local_state(local_tx, prev_latest_per_commitment_point, &Some(*delayed_payment_base_key), 0).0); // Accurate height doesn't matter as we're going to change 
```

Comment:
nit: excess newline at EOL.
Also, I found this comment very confusing. Maybe something more explicit like "We throw away the generated waiting_first_conf data as we aren't (yet) confirmed and we don't actually know what the caller wants to do. The data will be re-generated and tracked in check_spend_local_transaction if we get a confirmation."

Reply:
Updated